run.go

```diff
@@ -405,7 +438,8 @@ func runQuery(collection *mongo.Collection, method string, stages []interface{})
 			return nil, fmt.Errorf("fail to run update: %v", err)
 		}
 		cursor, err = collection.Find(context.Background(), bson.M{})
-
+	case getIndexesMethod:
```

Comment:
Do you want to be able to run this type of query from front-end, or is this just for testing purpose ?

Reply:
I made it as a quick workaround for testing and don't think it has to be available for end-user. The final decision is up to you as a maintainer.

Reply:
ok, fine for me. Allowing new method in the front-end requires a few changes, and I don't think it's worth the trouble